### PR TITLE
Switch to MIT license

### PR DESCRIPTION
**Pull request by jdhoffa**
Before merging this PR, we will need approval from all contributors.

### CODE REVIEW

**Review by maurolepore — CHANGES_REQUESTED** [review]
DESCRIPTION still mentions the old license. usethis::use_mit_license() should do the trick.

**Comment by maurolepore** [comment]
I approve changing to MIT license.

Let me know if you want me to direct the other authors to this PR:

```
Authors@R: 
    c(person(given = "Jackson",
             family = "Hoffart",
             role = c("aut", "cre"),
             email = "jackson.hoffart@gmail.com",
             comment = c(ORCID = "0000-0002-8600-5042")),
      person(given = "Mauro",
             family = "Lepore",
             role = c("aut", "ctr"),
             email = "maurolepore@gmail.com",
             comment = c(ORCID = "0000-0002-1986-7988")),
      person(given = "Klaus",
             family = "Hogedorn",
             role = "aut",
             email = "klaus@2degrees-investing.org"),
      person(given = "2 Degrees Investing Initiative",
             role = c("cph", "fnd"),
             email = "contact@2degrees-investing.org"))
```

**Comment by jdhoffa** [comment]
I approve the change to MIT license.

**Comment by maurolepore** [comment]
I support changing the license of this package to MIT.

**Comment by maurolepore** [comment]
@jdhoffa / @kla, do you have any idea how we should get the support from 2DII as an institution? And considering that -- in this context -- the three of us represent 2DII, Do we need any other consent than the one we just gave?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,2 @@
+YEAR: 2020
+COPYRIGHT HOLDER: 2 Degrees Investing Initiative France
```
Comment (maurolepore) [inline LICENSE:2]: Please confirm the word "France" -- I expected this to be 2DII in general.

**Review by maurolepore — CHANGES_REQUESTED** [review]
Please run the usethis helper, as I think a file is missing (LICENSE.md).

File: LICENSE (hunk 1 above)
Comment (jdhoffa) [reply LICENSE:2]: yes we need the word France

**Review by maurolepore — APPROVED** [review]

**Comment by maurolepore** [comment]
2DII directors confirmed they also support the change to MIT license.

**Pull request merged**